rpc/json/rpc.json

```diff
@@ -413,6 +413,22 @@
             }
         ]
     },
+    {
+        "description": "Returns sync state of this client",
+        "method": "sync_state",
+        "module": "net",
+        "params": [],
+        "result": {
+            "best_known_block_number": "0x248623",
+            "fast_time": "0x3e8",
+            "ibd": false,
+            "inflight_blocks_size": "0x0",
+            "low_time": "0x5dc",
+            "normal_time": "0x4e2",
+            "orphan_block_pool_size": "0x0"
+        },
+        "skip": true
+    },
```

Comment:
Please add a `returns` section to describe the returning data fields:
https://github.com/nervosnetwork/ckb/blob/develop/rpc/README.md#returns-1